packages/compass-saved-aggregations-queries/src/stores/aggregations-queries-items.ts

```diff
@@ -1,5 +1,102 @@
-function reducer(state = {}): typeof state {
+import { remote } from 'electron';
+import { join } from 'path';
+import fs from 'fs';
+import { AnyAction } from 'redux';
+import { FavoriteQueryCollection } from '@mongodb-js/compass-query-history';
+
+import { actions } from '../actions/aggregations-queries-actions';
+interface Item {
+  id: string;
+  name: string;
+  namespace: string;
+  lastModified: number;
+}
+
+export interface Aggregation extends Item {
+  pipeline: unknown[];
+}
+
+export interface Query extends Item {
+  collation: Record<string, unknown>;
```

Comment:
question: how do we plan to use this info? Is it for preview?

Reply:
This list will have a set of filters and depending on how we filter, we need info from Queries/Aggregations. 
We can actually remove this in this PR and once we get to filters, add more information.

Reply:
That's a good idea yeah, so we don't end up with stuff that won't be used